src/main/java/com/gooddata/executeafm/result/ExecutionResult.java

```diff
@@ -83,29 +83,29 @@ public Paging getPaging() {
     }
 
     /**
-     * @return attribute header items, for each attribute in each dimension, there is a list of headers
+     * @return header items, for each attribute or measure in each dimension, there is a list of headers
      */
-    public List<List<List<AttributeHeaderItem>>> getAttributeHeaderItems() {
-        return attributeHeaderItems;
+    public List<List<List<ResultHeaderItem>>> getHeaderItems() {
```

Comment:
This breaks backward compatibility. But I guess we have no choice.

Reply:
yes I know, since nobody uses that API yet and there is noservice in SDK, we agreed with @liry that it's fine to rollout those changes as bugfixes